src/Microsoft.PowerShell.GraphicalHost/commandHelpers/ShowCommandHelper.cs

```diff
@@ -183,7 +183,7 @@ Function GetParameterInfo
                 $validateSetAttributes = $parameter.Attributes | Where {
                     [ValidateSet].IsAssignableFrom($_.GetType())
                 }
-                if (($validateSetAttributes -ne $null) -and ($validateSetAttributes.Count -gt 0))
+                if (($null -ne $validateSetAttributes) -and ($validateSetAttributes.Count -gt 0))
```

Comment:
Good. Although if I had written the code myself I would have rather declared `$validateSetAttributes` as `[System.Array]` because then one would not only avoid the problem that the variable can be null but also prevents gotchas where in the case of array size 1 the type would not be an array and therefore addressing it later as $validateSetAttributes[0] would result in undesired behaviour (in most cases PowerShell would return the first character of the string representation). Just my two cents.